src/core/lib/iomgr/error.h

```diff
@@ -136,13 +136,15 @@ typedef enum {
   GRPC_ERROR_TIME_CREATED,
 } grpc_error_times;
 
+
 /// The following "special" errors can be propagated without allocating memory.
 /// They are always even so that other code (particularly combiner locks) can
 /// safely use the lower bit for themselves.
 
 #define GRPC_ERROR_NONE ((grpc_error *)NULL)
 #define GRPC_ERROR_OOM ((grpc_error *)2)
 #define GRPC_ERROR_CANCELLED ((grpc_error *)4)
+#define GRPC_ERROR_INTERNAL ((grpc_error *)8)
```

Comment:
We make the carve out that the lower bit is always usable in a comment just above:
```
/// They are always even so that other code (particularly combiner locks) can
/// safely use the lower bit for themselves.
```